modules/addbook.js

```diff
@@ -0,0 +1,86 @@
+class Book {
+  constructor(formElem, bookCollectionName, elemToInsertData) {
+    this.bookCollectionName = bookCollectionName;
+    this.elemToInsertData = elemToInsertData;
+    this.formElem = document.querySelector(formElem);
+    this.bookCollection = JSON.parse(localStorage.getItem(this.bookCollectionName)) || [];
+    this.bookTitleField = document.querySelector('.book-title');
+    this.bookAuthorField = document.querySelector('.book-author');
+    this.formEventHandler();
+    this.displayBooks();
+  }
+
+  updateLocalStorageData() {
+    // This method will update data in localStorage
+    localStorage.setItem(this.bookCollectionName, JSON.stringify(this.bookCollection));
+  }
+
+  addBookToList(title, author) {
+    // This method will add a new book into the booklist
+    this.bookCollection.push({ title, author });
+    this.updateLocalStorageData();
+    this.displayBooks();
+  }
+
+  removeBookFromList(index) {
+    // This method will remove the selected book from the booklist
+    this.bookCollection.splice(index, 1);
+    this.updateLocalStorageData();
+    this.displayBooks();
+  }
+
+  generateHtmlForBookList() {
+    // This method will generate HTML element for the complete booklist
+    let boilerPlate = '';
+    this.bookCollection.forEach((book, index) => {
+      boilerPlate += `<tr class="book-item">
+          <td>"${book.title}" by ${book.author}</td>
+          <td>
+            <button class="btn btn-light" data-index=${index}>
+              Remove
+            </button>
+          </td>
+        </tr>`;
+    });
+    return boilerPlate;
+  }
+
+  activateRemoveButton() {
+    // This method will add & activate remove event on remove button in each book row
+    const table = document.querySelector('.booklist-table');
+    const btn = table.querySelectorAll('button');
+    btn.forEach((btnR, index) => {
+      btnR.addEventListener('click', () => {
+        this.removeBookFromList(index);
+      });
+    });
+  }
+
+  displayBooks() {
+    // This method will generate the complete booklist and display it
+    const bookList = document.querySelector(this.elemToInsertData);
+    bookList.innerHTML = this.generateHtmlForBookList();
+    this.activateRemoveButton();
+    this.formInputFieldNormalize();
+  }
```

Comment:
- [ ] You have done a great job displaying the book :clap: However the books do not display when reload the page. Kindly make sure the books saved in the localstore are display when the pages loaded.

**HIT:**
```js
document.addEventListener("DOMContentLoaded", () => {
  // your code here
});
```